identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -584,3 +584,11 @@ RmFpbGVkIEJhY2t1cCBDb2RlIEF0dGVtcHRz=Failed Backup Code Attempts
 # SAAS Product taglines
 saas.tagline=Create seamless login experiences for your applications in minutes
 saas.slogan={{productName}} is an IDaaS empowering developers to implement secure and frictionless access.
+
+# Duo Authenticator
+error.failed.with.duo=Failed Authentication with Duo
+error.user.not.registered=Couldn't get the user information. User may not be registered in Duo.
+error.duo.user.not.found=Couldn't get the verified user name.
+error.mobile.not.found=User may not have a mobile number in user's profile for Duo Authentication. Please update it.
```

Comment:
This error message kind of give a wrong message. Because in the first sentence it says user "may not" have a number, and in the second sentence it says to update it.